lv_draw/lv_draw_rbasic.c

```diff
@@ -313,10 +313,11 @@ void lv_rmap(const lv_area_t * cords_p, const lv_area_t * mask_p,
 
                     lv_rpx(col, row, mask_p, recolored_px, LV_OPA_COVER);
                 } else {
-                    lv_rpx(col, row, mask_p, *px_color, LV_OPA_COVER);
+                    //lv_rpx(col, row, mask_p, *px_color, LV_OPA_COVER);
```

Comment:
Please just delete this. No need to leave ghosts of past code hanging around in comment. That would get messy fast, and we always have git if we ever need to refer back to how it used to work before this mod.